Formula/dependency-check.rb

```diff
@@ -1,9 +1,9 @@
 class DependencyCheck < Formula
   desc "OWASP Dependency Check"
   homepage "https://www.owasp.org/index.php/OWASP_Dependency_Check"
-  url "https://dl.bintray.com/jeremy-long/owasp/dependency-check-1.4.3-release.zip"
-  version "1.4.3"
-  sha256 "53ce0904475897cb79a4ef162fe6e9068f8c23a788d8855bb46fe059cf8664c3"
+  version "1.4.4"
+  url "https://dl.bintray.com/jeremy-long/owasp/dependency-check-#{version}-release.zip"
```

Comment:
Please leave the `url` above the `version`, and hardcode the number into it.  That makes it easier for new contributors to read.